container/controller.go

```diff
@@ -693,7 +695,7 @@ func (c *Controller) checkPrereqs(prereqsPassed chan bool, rpcDead chan struct{}
 			if !failedAny {
 				glog.Infof("Passed all prereqs.")
 				prereqsPassed <- true
-				return nil
+				break WaitingForPrereqs
```

Comment:
Our initial design for the prereqs is that they are only checked the first time we kick off the service, so this change would need some discussion.